CMakeLists.txt

```diff
@@ -279,8 +279,6 @@ if (HAVE_AVX2)
   list (APPEND SOURCEFILES ${fftw_dft_simd_avx2_SOURCE} ${fftw_rdft_simd_avx2_SOURCE})
 endif ()
 
-set (FFTW_VERSION 3.3.10)
```

Comment:
This variable is used in generated file - cmake.config.h.in and FFTW3ConfigVersion.cmake.in.

Reply:
Thanks @volo-zyko  for the review! I've changed the variables now!